frontend/src/components/profileList/Avatar.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+	<div class="c_media">
+		<a
+			href="#"
```

Comment:
I realized now that this should probably become a router-link element. I'll have a look later how to restructure the html.. (also for other A elements that actually go somewhere..)

Reply:
This works: _(you have to pass the id to the Avatar component)_

```vue
<router-link to="`/profiles/${id}`">
	<div
		class="c_media__asset"
		:class="{
			'c_asset--online': isOnline,
			'c_asset--offline': !isOnline,
		}"
	>
		<div class="c_asset__circle">
			<img :src="`/api/avatars/${avatar}`" alt="" />
		</div>
	</div>
</router-link>
```

Reply:
Discussed in person, routing via script instead of html element